charts/draft-store-service/values.yaml

```diff
@@ -0,0 +1,18 @@
+java:
+  applicationPort: 8800
+  environment:
+    DRAFT_STORE_DB_HOST: "{{ .Release.Name }}-postgresql"
+    DRAFT_STORE_DB_NAME: {{ .Values.java.postgresql.postgresqlDatabase}}
+    DRAFT_STORE_DB_USER_NAME: {{ .Values.java.postgresql.postgresqlUsername}}
+    DRAFT_STORE_DB_PASSWORD: {{ .Values.java.postgresql.postgresqlDatabase}}
```

Comment:
how does this work if you use a secret instead?
i.e in AAT won't this still show up as an environment variable?

Reply:
It will be same as how it is today. Password will still be managed by secrets mounted onto container.

Reply:
if you described the pod won't it show this password though? even though its loaded from secrets

Reply:
It doesn't work somehow, will see if i can get it working. But, as far as i have read, it will just take the value from computed values.yaml . So, in this case it will display "hmcts" and not the actual password. Anyway, there won't be a postgres pod in AAT . I don't think this will be harmful . I will anyway have a look when i get this to AAT.

Reply:
It works, Are u happy for me to try and see in AAT ?

Reply:
yea